near-sdk/src/store/lookup_set/mod.rs

```diff
@@ -54,6 +74,46 @@ where
     T: BorshSerialize + Ord,
     H: CryptoHasher<Digest = [u8; 32]>,
 {
+    fn lookup_key<Q: ?Sized>(prefix: &[u8], value: &Q, buffer: &mut Vec<u8>) -> LookupKey
+    where
+        Q: BorshSerialize,
+        T: Borrow<Q>,
+    {
+        // Concat the prefix with serialized key and hash the bytes for the lookup key.
+        buffer.extend(prefix);
+        value.serialize(buffer).unwrap_or_else(|_| env::panic_str(ERR_ELEMENT_SERIALIZATION));
+
+        H::hash(buffer)
+    }
+
+    fn contains_trie_element<Q: ?Sized>(prefix: &[u8], value: &Q) -> bool
+    where
+        Q: BorshSerialize,
+        T: Borrow<Q>,
+    {
+        let lookup_key = Self::lookup_key(prefix, value, &mut Vec::new());
+        env::storage_has_key(&lookup_key)
+    }
+
+    fn get_mut_inner<Q: ?Sized>(&mut self, value: &Q) -> &mut EntryState
+    where
+        T: Borrow<Q>,
+        Q: BorshSerialize + ToOwned<Owned = T>,
+    {
+        let prefix = &self.prefix;
+        //* ToOwned bound, which forces a clone, is required to be able to keep the value in the cache
+        let entry = self.cache.get_mut(value.to_owned());
+        entry.get_or_init(|| {
+            if Self::contains_trie_element(prefix, value) {
+                EntryState::Present
+            } else {
+                EntryState::Absent
+            }
+        });
+        let entry = entry.get_mut().unwrap_or_else(|| unreachable!());
```

Comment:
I see this is pulled from lookup map but this must have been missed when cleaning up these types of errors, this still includes file info in the code and error. Better to switch this to `env::abort`